tests/foreman/cli/test_rhcloud_inventory.py

```diff
@@ -264,3 +266,44 @@ def test_rhcloud_external_links():
 
     :CaseLevel: System
     """
+
+
+@pytest.mark.tier3
+def test_positive_generate_all_reports_job(target_sat):
+    """Generate all reports job via foreman-rake console:
+
+    :id: a9e4bfdb-6d7c-4f8c-ae57-a81442926dd8
+
+    :Steps:
+        1. Execute Foreman GenerateAllReportsJob via foreman-rake.
```

Comment:
looking at the bz, the reporter says the "Automatic inventory upload" setting needs to be set to "No" to manifest the issue, but I don't see the test doing that

Reply:
Good catch! Updated.